src/theory/theory_engine.h

```diff
@@ -277,11 +277,13 @@ class TheoryEngine {
    */
   void lemma(TrustNode node,
              theory::LemmaProperty p,
-             theory::TheoryId atomsTo = theory::THEORY_LAST,
              theory::TheoryId from = theory::THEORY_LAST);
 
-  /** Enusre that the given atoms are send to the given theory */
-  void ensureLemmaAtoms(const std::vector<TNode>& atoms, theory::TheoryId theory);
+  /** Ensure atoms from the given node are sent to the given theory */
+  void ensureLemmaAtoms(Node n, theory::TheoryId atomsTo);
```

Comment:
Can we make this a `TNode`?

```suggestion
  void ensureLemmaAtoms(TNode n, theory::TheoryId atomsTo);
```